Lib/test/test_typing.py

```diff
@@ -489,6 +489,66 @@ def test_ellipsis_in_generic(self):
         typing.List[Callable[..., str]]
 
 
+class LiteralTests(BaseTestCase):
+    def test_basics(self):
+        # All of these are allowed.
+        Literal[1]
+        Literal[1, 2, 3]
+        Literal["x", "y", "z"]
+        Literal[None]
+
+    def test_illegal_parameters_do_not_raise_runtime_errors(self):
+        # Type checkers should reject these types, but we do not
+        # raise errors at runtime to maintain maximium flexibility.
+        Literal[int]
+        Literal[Literal[1, 2], Literal[4, 5]]
+        Literal[3j + 2, ..., ()]
+        Literal[b"foo", u"bar"]
```

Comment:
What's illegal about this one? The PEP shows examples of both `b"..."` and `u"..."` (L167-168).

Reply:
Likely this and the one below were prohibited in an older version of the PEP or during discussion.